jib-gradle-plugin/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+- Support for local base images using a URL scheme in `jib.from.image`. Use the format `docker://name-of-image` to use a base image stored in the local Docker daemon, or `tar:///path/to/file.tar` to use a base image tarball stored on the filesystem ([#1468](https://github.com/GoogleContainerTools/jib/issues/1468), [#1905](https://github.com/GoogleContainerTools/jib/issues/1905))
```

Comment:
If `tar://` supports a relative path (if so, I think it should be relative to the project root), worth mentioning?